qiling/extensions/pipe.py

```diff
@@ -84,3 +101,82 @@ def flush(self) -> None:
 
     def writable(self) -> bool:
         return True
+
+class SimpleBufferedStream(TextIO):
+    """Simple buffered IO.
+    """
+
+    def __init__(self):
+        self.buff = bytearray()
+        self.cur = 0
+
+    def lseek(self, offset: int, origin: int) -> int:
```

Comment:
Same here, `lseek` is not part of the interface; could use `seek` instead.

Reply:
Fine, I agree we can change the implementation of lseek syscall to use seek instead,

Reply:
Removed.